src/content/docs/style-guide/writing-strategies/nav-best-practices.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Docs nav best practices
+metaDescription: "Everything I know about information architecture I learned in library science school."
+---
+
+What is the purpose of a navigation sidebar menu? This is worth considering, because this reader experience is pretty ubiquitous across doc sites, and things that are everywhere tend not to be considered much.
+
+I think a nav menu serves two fundamental purposes:
+
+1. Help readers find the information they're looking for
+2. Give readers insight into the conceptual model of a body of knowledge (in our case, a conceptual model of New Relic)
+
+## Find information [#find]
+
+When I imagine someone finding information, I find that my initial impulse is to picture someone moving in a straight line from question to answer. I feel this impulse, even though I know it to be false!
+
+When human beings move through collections of information—whether it's a docs site, a bookstore, or a temporary collection of information created from an internet search—they do so in a meandering, inefficient, and emotionally impulsive way. Although this process is inefficient, it's the primary way that human beings build conceptual models of understanding!
+
+The ground-breaking library scientist, Marcia Bates, described this organic process of information gathering as [berrypicking](https://pages.gseis.ucla.edu/faculty/bates/berrypicking.html). In her model, **browsing** is what people do when they're trying to make sense of a poorly understood body of knowledge. 
+
+As I see it, the left nav menu is **primarily a tool for browsing**.
+
+## Conceptual model [#model]
+
+When our conceptual model of New Relic is confusing to us, our nav is going to reflect that. This does a disservice both to our readers and ourselves.
+
+When we look at the nav and are confused by how its organized, this is a fundamentally useful clue to us (the docs team) that we need to work on our conceptual model. 
+
+As a reader browses through our nav, are they picking up on key terms? Are they starting to get a sense of the shape of New Relic as a system? Are they able to build enough of a mental model to take action in a meaningful way?
```

Comment:
This feels a little open-ended and incomplete. I want more from this. I want someone to tell me what to do! How can I make decisions that are more meaningful? Perhaps that's not answerable here, but I'd prefer to end this paragraph with a statement rather than a question.